manifests/init.pp

```diff
@@ -66,5 +51,19 @@
     ensure  => present,
   }
 
-  create_resources(cron, $policies_data)
+  file { 'foreman_scap_client_cron':
+    path    => '/etc/cron.d/foreman_scap_client_cron',
+    content => template('foreman_scap_client/cron.erb'),
+    owner   => 'root',
+    ensure  => present,
+  }
+
+  # Remove crons previously installed here
+  $policies_cron = inline_template('<%= policies.map { |p|
```

Comment:
I'm afraid this won't work. Let's assume there are two policies for a host with ids 1 and 2. Then later, 1 is removed so in ENC data we have only 2. This would then only clear 2 right? I think we should clean all policies we created and that were removed prior this patch. That's why I suggested shell script executed through exec resource that would grep for all crontab entries with foreman_scap_client in their name and remove them. It should be using "onlyif" so it's active only if grep finds such record, otherwise it would always generate active puppet runs.